opengever/maintenance/scripts/fix_mail_class_intids.py

```diff
@@ -0,0 +1,105 @@
+"""
+A bin/instance run script which checks and fixes the class of
+all mail objects that are references from from the intid catalog.
+
+bin/instance0 run fix_mail_class_intids.py
+"""
+
+from Acquisition import aq_base
+from Acquisition import aq_parent
+from ftw.mail.mail import Mail
+from opengever.mail.mail import OGMail
+from opengever.maintenance.debughelpers import setup_debug_mode
```

Comment:
`setup_debug_mode` shouldn't be used in scripts like this. It's intended for exactly what it says, a helper for setting up the `bin/instance debug` environment. Please use [`setup_app()`](https://github.com/4teamwork/opengever.maintenance/blob/master/opengever/maintenance/debughelpers.py#L56) and [`setup_plone_site()`](https://github.com/4teamwork/opengever.maintenance/blob/master/opengever/maintenance/debughelpers.py#L31) as needed.